num_of_units: 4
unit_0:
  x: 0.175474
  y: -0.117232
  z: 0.014003
  p0: -0.180852
  p1: 0.848461
unit_1:
  x: 0.177537
  y: 0.117540
  z: 0.010078
  p0: 0.220078
  p1: 0.830603
unit_2:
  x: -0.167694
  y: 0.110225
  z: 0.009217
  p0: 0.394748
  p1: 0.866774
unit_3:
  x: -0.171625
  y: -0.102814
  z: 0.006353
  p0: 0.289890
  p1: 0.865115
